swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/repositories/InMemoryTaskTrackingRepository.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ *
+ *  * Copyright 2018 Netflix, Inc.
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License")
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *   http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.swabbie.repositories
+
+import java.time.Clock
+import java.util.concurrent.TimeUnit
+
+class InMemoryTaskTrackingRepository(
```

Comment:
For testing but probably not necessary.

Reply:
For all repositories like this, I'd recommend adding a logger and an init that reports what repository is being used.

```
init {
  log.info("Using ${javaClass.simpleName}")
}
```